rotkehlchen/tests/data/rotki_generic_events.csv

```diff
@@ -0,0 +1,8 @@
+ID,Type,Location,Currency,Amount,Fee,Fee Currency,Description,Timestamp
+1xyz,Deposit,kucoin,EUR,1000.00,,,Deposit EUR to Kucoin,2022-07-27 09:00:00
+2xyz,Withdrawal,binance,USDT,99.00,1.00,USDT,,2022-07-28 09:00:00
+3xyz,Withdrawal,kraken,BNB,1.01,,,,2022-07-29 09:00:00
+4xyz,Staking,luno,ETH,0.0513,,,ETH Staking reward from QRS,2022-08-01 09:00:00
+5xyz,Loss,coinbase,BTC,0.0910,,,,2022-08-02 09:00:00
+6xyz,Income,cex,DAI,1000.00,,,,2022-08-03 09:00:00
+7xyz,Invalid,bisq,BCH,0.3456,,,,2022-08-05 09:00:00
```

Comment:
From the user side how are these ids going to be handled? Are they needed or can be randomly created by us?

Reply:
They're needed. If we randomly generate them, testing becomes difficult. 
Otherwise, randomly generating an event identifier works better.

Reply:
Code and UX should come first. How is testing more complicated?

Reply:
>  How is testing more complicated?

We'd be unable to know what the `event_identifier` is when comparing `expected_history_events` to `history_events` as we generate it randomly.
Unless I create an `assert_history_event_is_equal(history_event, expected_history_event)` function